tools/sync_test_server/Dockerfile

```diff
@@ -5,8 +5,8 @@ ARG ROS_DE_VERSION
 # Add realm repo
 RUN apt-get update -qq \
     && apt-get install -y curl npm \
-    # && curl -s https://packagecloud.io/install/repositories/realm/realm/script.deb.sh \
-    && curl -s https://packagecloud.io/install/repositories/realm/realm-testing/script.deb.sh | bash
+    && curl -s https://packagecloud.io/install/repositories/realm/realm/script.deb.sh | bash
```

Comment:
Yes. Testing releases are not getting correctly deployed, so the stable release is the latest available one.